.drone.yml

```diff
@@ -279,12 +281,12 @@ steps:
 
 trigger:
   branch:
-    - "staging"
-    - "staging-next"
+    exclude:
+      - "production"
+      - "test"
+      - "levelbuilder"
```

Comment:
Aaaactually... should we only do this on a push to `staging` / `staging-next` ? Is that the intended behavior... that it is a cache of what `staging` looks like?

Reply:
I'm not confident in what this does to be sure.

Reply:
I *think* it makes sense to still use the cache of staging on branches off branches; the idea is that rather than cloning the entire repro from scratch, we restore a cache of what's on staging and then just fetch whatever deltas are necessary for the specific branch we're looking at. Even branches off branches off staging should benefit from that.

Reply:
Changing this trigger might be causing us to cache staging a lot more often. Notice the missing `event`

Reply:
Oh, lol, no; @wilkie you were absolutely right! This isn't the step that *restores* the cache, it's the step that *creates* the cache